db/migrate/20150207064616_create_works.rb

```diff
@@ -0,0 +1,13 @@
+class CreateWorks < ActiveRecord::Migration
+  def change
+    create_table :works do |t|
+      t.string :isbn
```

Comment:
A work can have multiple ISBN numbers associated with it, not just the
short and long versions of the same number... 10 digit and 13 digit?
I'm thinking of some of the early books when ISBN was still coming into its
own: Apple publishers and Scholastic were notorious for putting barcodes on
the book which are neither of the two official ISBNs.  Each time I
encounter something like this, I am tempted to swear.

On Tue, Feb 10, 2015 at 10:15 PM, Aaron Panchal <notifications@github.com>
wrote:

> In db/migrate/20150207064616_create_works.rb
> <https://github.com/hammersmith/bookends/pull/7#discussion_r24476095>:
>
> > @@ -0,0 +1,13 @@
> > +class CreateWorks < ActiveRecord::Migration
> > +  def change
> > +    create_table :works do |t|
> > +      t.string :isbn
>
> We should probably have two columns: one for each type of isbn
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/hammersmith/bookends/pull/7/files#r24476095>.
>



-- 

​--------
​-------===============########===============---------------​


Want to quickly check
​a ​
system status
​ or​
report
​a problem to the IT team?
Use http://justme.westmont.edu
​
​
Have a problem that requires tracking and IT email response?
Use http://mayday.westmont.edu​